src/styles/main.scss

```diff
@@ -0,0 +1,57 @@
+$width-box: 100px;
+$gap: 1px;
+$days-of-week: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
```

Comment:
There is a syntax error in the `$days-of-week` map. The correct syntax for defining a map in SCSS is to use colons to separate keys and values, like this: `(mon: 0, tue: 1, ...)`.